src/main/java/org/symphonyoss/symphony/messageml/markdown/nodes/SelectNode.java

```diff
@@ -0,0 +1,31 @@
+package org.symphonyoss.symphony.messageml.markdown.nodes;
+
+import org.commonmark.node.CustomBlock;
+
+public class SelectNode extends CustomBlock {
+  private final static String LEAD = "Dropdown:";
```

Comment:
Do you mean to have no space after the delimiter (e.g. "_(Dropdown:complete-required-id):_")?

Reply:
As we talked on 1-1, this fits the Spec.